src/common_swagger_api/schema/tools.clj

```diff
@@ -62,6 +62,22 @@
 (def ToolDetailsDocs
   "This endpoint returns the details for one tool accessible to the user.")
 
+(def ToolInstallRequestSummary "Request Tool Installation")
```

Comment:
We already had a `ToolRequestSummary` schema defined (used as part of this `GET /tool-requests` endpoint response schema), so I named the `:summary` string for this endpoint  `ToolInstallRequestSummary`, and the rest of these `:summary` and `:description` strings follow suit.